src/infra/flow-system-inmem/tests/tests/utils/task_driver.rs

```diff
@@ -30,6 +30,8 @@ pub(crate) struct TaskDriverArgs {
     pub(crate) dataset_id: Option<DatasetID>,
     pub(crate) run_since_start: Duration,
     pub(crate) finish_in_with: Option<(Duration, TaskOutcome)>,
+    pub(crate) max_slice_size: Option<u64>,
+    pub(crate) max_slice_records: Option<u64>,
```

Comment:
Why is this a part of task driver? We should define parameters at flow configuration level only.

Reply:
It's here just to assert we are using the right parameters. Not sure I can do it in another way

Reply:
Changed to assert logical plan